### DISCUSSION TITLE
App version 1.027 released - with Math Channels

### DISCUSSION THREAD

**Discussion by fhdm-dev** [fhdm-dev/scoppy#120] [Announcements]
Version 1.027 of the Scoppy Android app has just been released.

The only new feature introduced in this release is math channels.

To use, tap Menu then Math and then Edit.
There is some [basic documentation](https://oscilloscope.fhdm.xyz/app-help/math) and also an [article on hackster.io](https://www.hackster.io/sandy-roberts2/scoppy-oscilloscope-math-channels-tutorial-50abc0) that shows some examples.

Enjoy :smiley:

**Comment by cheadrian** [comment]
Is there a way to add a math function to average readings for cutting out noise, or a noise filter setting.

**Reply by fhdm-dev** [reply]
Hi,

It's not possible with the existing math functionality - but I might add support for this feature in future versions of the app.
